engine/glfw/js/library_glfw.js

```diff
@@ -444,7 +444,9 @@ var LibraryGLFW = {
         if (GLFW.mouseWheelFunc) {
           {{{ makeDynCall('vi', 'GLFW.mouseWheelFunc') }}}(GLFW.wheelPos);
         }
-        event.preventDefault();
+        if (event.cancelable) {
```

Comment:
Checking if the event can be cancelled before calling preventDefault() solves half of the problem.